app/decisions/get.controller.js

```diff
@@ -0,0 +1,28 @@
+const { logger } = require('../../common/logging/logger')
+const { getSentencePlanComments } = require('../../common/data/sentencePlanningApi')
+const { getCommentText } = require('../../common/utils/getCommentText')
+
+const getDecisions = async (
+  { path, errors, errorSummary, body, renderInfo, params: { planId }, session: { 'x-auth-token': token } },
+  res
+) => {
+  const renderDetails = renderInfo || {}
+  let comments = ''
+  renderDetails.backurl = path.substring(0, path.lastIndexOf('/'))
```

Comment:
Is there enough commonality to warrant a helper that auto fills back and next urls in the `renderDetails` object?

Reply:
Probably is - we've already raised a ticket for determining back/next from a common place so we can address it there